src/modelldcatnotordf/modelldcatno.py

```diff
@@ -444,9 +447,15 @@ def to_rdf(self, format: str = "turtle", encoding: Optional[str] = "utf-8") -> s
         """
         return self._to_graph().serialize(format=format, encoding=encoding)
 
-    def _to_graph(self) -> Graph:
+    def _to_graph(
+        self, type: str = MODELLDCATNO.Property, selfobject: Any = None
```

Comment:
Det er vanskelig å forstå hva `selfobject` er. Er det mulig å gi argumentet et bedre navn?

Reply:
Jeg har ikke noe bedre forslag egentlig :thinking: , men jeg er åpen for bedre forslag. Det er et objekt av typen Any, det kan være enten en Bnode eller en URI, alt ettersom vi har mangler eller har identifier for klassen..